lib/generators/rspec/mailer/templates/preview.rb

```diff
@@ -5,7 +5,7 @@ class <%= class_name %>Preview < ActionMailer::Preview
 
   # Preview this email at http://localhost:3000/rails/mailers/<%= file_path %>/<%= action %>
   def <%= action %>
-    <%= class_name %><%= Rails.version.to_f >= 5.0 ? "Mailer" : "" %>.<%= action %>
+    <%= Rails.version.to_f >= 5.0 ? class_name.sub(/(Mailer)?$/, 'Mailer') : class_name %>.<%= action %>
```

Comment:
I feel like we should have a spec for this

Reply:
Ok, I'll work on 👍

Reply:
@JonRowe 
I wrote test 😄 
please review again.

Reply:
Thanks